docs/developing/developing_secrets.md

```diff
@@ -0,0 +1,47 @@
+---
+
+copyright:
+years: 2021
+lastupdated: "2021-08-16"
+
+---
+
+{:new_window: target="blank"}
+{:shortdesc: .shortdesc}
+{:screen: .screen}
+{:codeblock: .codeblock}
+{:pre: .pre}
+{:child: .link .ulchildlink}
+{:childlinks: .ullinks}
+
+# Hello world with a secret
+{: #secrets}
+
+This example helps you learn how to develop an {{site.data.keyword.edge_service}} that uses secrets. You use secrets to ensure that login credentials and other sensitive information are kept secure
```

Comment:
Consider changing: 

This example helps you learn how to develop an {{site.data.keyword.edge_service}} that uses secrets. You use secrets to ensure that login credentials and other sensitive information are kept secure

to:

This example teaches you how to develop an {{site.data.keyword.edge_service}} that uses secrets. Secrets ensure that login credentials and other sensitive information are kept secure.

Reply:
done